views/sample_tracking_view.sql

```diff
@@ -6,11 +6,19 @@ WITH sample_events AS (
     WHERE role_type = 'sample'
       AND event_type IN ('sample_manifest.updated', 'labware.received', 'library_start', 'library_complete', 'sequencing_start', 'sequencing_complete', 'order_made')
       AND occured_at >= DATE_SUB(NOW(), INTERVAL 2 YEAR)
+),
+labware_manifest_crated_event AS (
+    SELECT event_type, occured_at ,subject_friendly_name
+    FROM [events].flat_events_view
+    WHERE role_type = 'labware'
+    AND event_type =  'sample_manifest.created'
+    AND occured_at >= DATE_SUB(NOW(), INTERVAL 2 YEAR)
 )
 
 SELECT
     sample_flowcell_view.study_name,
     sample_flowcell_view.id_study_lims AS study_id,
+    MIN(IF(labware_manifest_crated_event.event_type, labware_manifest_crated_event.occured_at, NULL)) manifest_created,
```

Comment:
I think this is supposed to say `IF(labware_manifest_crated_event.event_type='sample_manifest.created', ...`
but you don't need an `IF` here at all, because the subquery only includes one event type.